test/Riok.Mapperly.Tests/TestSourceBuilder.cs

```diff
@@ -38,13 +39,18 @@ public partial class Mapper
 
     private static string BuildAttribute(TestSourceBuilderOptions options)
     {
-        var attrs = new[]
+        var attrs = new List<string>
         {
             Attribute(options.UseDeepCloning),
             Attribute(options.ThrowOnMappingNullMismatch),
-            Attribute(options.ThrowOnPropertyMappingNullMismatch),
+            Attribute(options.ThrowOnPropertyMappingNullMismatch)
         };
 
+        if (options.PropertyNameMappingStrategy != PropertyNameMappingStrategy.CaseSensitive)
+        {
+            attrs.Add($"PropertyNameMappingStrategy=PropertyNameMappingStrategy.{options.PropertyNameMappingStrategy}");
```

Comment:
To match the other attribute key value pairs, I'd add a space before and after the equals sign. Maybe you could also use the `nameof` operator instead of the fixed string.

Reply:
you are right!